dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerFactoryImpl.java

```diff
@@ -1183,4 +1217,21 @@ public void updateUserReferences(String userId, String replacementUserId)throws
         }
 	}
 
+	/**
+	 * Validates if a string is a valid velocity variable name for content types.
+	 * Variable names should start with a letter, contain only alphanumeric characters,
+	 * underscores, or hyphens, and have a reasonable length limit.
+	 *
+	 * @param variableName the variable name to validate
+	 * @return true if the variable name is valid, false otherwise
+	 */
+	private boolean isValidVariableName(final String variableName) {
+		if (!UtilMethods.isSet(variableName)) {
+			return false;
+		}
+		// Variable names should start with letter, contain alphanumeric, underscore, hyphen
+		// and be between 1-255 characters for reasonable limits
+		return variableName.matches("^[a-zA-Z][a-zA-Z0-9\\-_]{0,254}$");
+	}

```

Comment:
have a look at this https://github.com/dotCMS/core/blob/6eac100e716a99e1c0f14c6d699bb26a5d08a36e/dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowFactoryImpl.java#L93

Reply:
you also have a compilation error
home/runner/work/core/core/dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerFactoryImpl.java:[1008,5] error: unreported exception DotSecurityException; must be caught or declared to be thrown

Reply:
fixed - pls check again, let me know if anything!

Reply:
agree, see my comments above.

Reply:
pls check now